database-attachments.md

```diff
@@ -149,3 +149,18 @@ Display the uploaded file on a page:
     }
 
     <img src="<?= $featuredImage ?>" alt="Featured Image">
+
+<a name="inverse-of-relation"></a>
+#### The inverse of the relation
+
+If you need to access the owner of a file, you can use the `attachment` property of the `File` model:
+
+    public $morphTo = [
+        'attachment' => []
+    ];
+    
+Example:  
+
+    $user = $file->attachment;
+    
+For more information read the [polymorphic relationships](https://octobercms.com/docs/database/relations#polymorphic-relations)
```

Comment:
Please use relative linking, not absolute. I.e. `[deferred binding](../database/relations#deferred-binding)`